Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:trusty
+
+# Get the basic VIC dependencies
+RUN apt-get update -q && \
+    apt-get install -y -q --no-install-recommends \
+        ca-certificates \
+        build-essential \
+        ssh \
+        netcdf-bin \
+        libnetcdf-dev \
+        libopenmpi-dev \
+        openmpi-bin \
```

Comment:
Very cool - with Docker we also get around the "compiling with mpi challenge" mentioned in issue #375